main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -429,53 +423,10 @@ DataTableView.prototype._renderDataTables = function(table, headerTable) {
   });
 };
 
-DataTableView.prototype._adjustDataTables = function() {
```

Comment:
There are a number of references to this function which need to be updated, both in this module and in `cell-ui.js`

Reply:
Done